mkdocs.yml

```diff
@@ -0,0 +1,32 @@
+site_name: Project Name
```

Comment:
Maybe we should have a 'Project Name' find and replace thing. We currently only have `skeleton-project` which is more a project slug. Just a thought.